### PR TITLE
[LS-100]-Module1-Micro-build-Part2 (by forge)

### PR DESCRIPTION
**Pull request by MONICABLANCO**
Descritption: Module 1 is built as required in: (https://everfi.atlassian.net/browse/LS-100) and according to: (https://www.figma.com/file/J7Q4uLlvVdkM0HLyLlUNsr/WCN---(LS)-Low-Fidelity-Wireframes?node-id=190%3A10027) and Assets provided.
* Pages 7-12: build
* Minor fixes on previous build

Disclaimer: Asstes-images are added as placeholders until real ones are provided.

### CODE REVIEW

File: app/modules/m01/a01/page-10/content.yml [hunk 1]
```diff
@@ -1,3 +1,187 @@
...
-  - type: container
-    blocks: []
+  - type: component
+    path: vertical-split
+    parameters:
+      maxwidth: true
+      primary:
+        enable: false
+      asset:
+        enable: false
```
Comment (senugeorge) [inline app/modules/m01/a01/page-10/content.yml:9]: Why the full-bleed-card component was not used? There is no vertical-split design here.

File: app/modules/m01/a01/page-11/content.yml [hunk 2]
```diff
@@ -1,3 +1,51 @@
...
+    parameters:
+      maxwidth: true
+      primary:
+        enable: true
+        background:
+          type: color
+          color: 'rgba(245,245,245,1)'
+        columns: 6
+      asset:
+        enable: false
```
Comment (senugeorge) [inline app/modules/m01/a01/page-11/content.yml:13]: Same here, horizontal-split

File: app/modules/m01/a01/page-12/content.yml [hunk 3]
```diff
@@ -1,3 +1,82 @@
...
+    parameters:
+      maxwidth: true
+      primary:
+        enable: true
+        background:
+          type: color
+          color: 'rgba(245,245,245,1)'
+        columns: 6
+      asset:
+        enable: false
```
Comment (senugeorge) [inline app/modules/m01/a01/page-12/content.yml:13]: Horizontal-split here too!

File: app/modules/m01/a01/page-7/content.yml [hunk 4]
```diff
@@ -1,3 +1,59 @@
...
+        enable: true
+        class: m-stack-none
+        background:
+          type: color
+          color: 'rgba(245,245,245,1)'
+        columns: 6
+      asset:
+        enable: false
+      columns: 12
+      header: vertical-split_parameters_header_1
```
Comment (senugeorge) [inline app/modules/m01/a01/page-7/content.yml:16]: Same here... horizontal-split can be used

File: app/modules/m01/a01/page-5/content.yml [hunk 5]
```diff
@@ -63,7 +63,7 @@ blocks:
                       container_class: m-stack-sm
                       parameters:
                         callouts:
-                          - callout: generic
+                          - callout: informational
```
Comment (senugeorge) [inline app/modules/m01/a01/page-5/content.yml:66]: horizontal-split on this page as well...

**Review by senugeorge — CHANGES_REQUESTED** [review]

**Review by senugeorge — APPROVED** [review]

File: app/modules/m01/a01/page-10/content.yml (hunk 1 above)
Comment (MONICABLANCO) [reply app/modules/m01/a01/page-10/content.yml:9]: Needed to set Navigation buttons to both margings of page.

**Pull request merged**